web/src/Sidebar.scss

```diff
@@ -29,25 +32,62 @@
   list-style: none;
   margin: 0;
   padding: 0;
+  border-bottom: 1px solid $color-navy-light;
+}
+
+.Sidebar-toggle {
+  position: sticky;
+  bottom: 0;
+  background-color: $color-navy;
+  border: 0 none;
+  border-top: 1px solid $color-navy-light;
+  width: 100%;
+  color: inherit;
+  font-size: inherit;
+  font-family: inherit;
+  display: flex;
+  align-items: center;
+  height: $sidebar-item;
+  margin: 0;
+  padding: 0;
+}
+
+.Sidebar-toggle svg {
+  fill: $color-white;
+  width: $spacing-unit * 1.5;
+  transform: rotate(180deg);
+}
+
+.Sidebar.is-closed .Sidebar-toggle svg {
+  transform: rotate(0);
+  transition-property: transform;
```

Comment:
should the transition fields be on the plain .Sidebar-toggle svg selector above?